.github/workflows/master.yml

```diff
@@ -7,40 +7,39 @@ jobs:
   build:
 
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
required?

Reply:
No.  If it is `false`, when some LTS versions fail, other versions will be cancelled. 
https://github.com/ishiy1993/haiji/actions/runs/3087777392